ui/pages/settings/networks-tab/network-form/network-form.component.js

```diff
@@ -451,28 +499,26 @@ export default class NetworkForm extends PureComponent {
     const { networksToRender } = this.props;
     const { chainId: stateChainId } = this.state;
     const isValidUrl = validUrl.isWebUri(url);
-    const chainIdFetchFailed = this.hasError(
-      'chainId',
-      t('failedToFetchChainId'),
-    );
+    const chainIdFetchFailed = this.hasError('chainId', 'failedToFetchChainId');
     const [matchingRPCUrl] = networksToRender.filter((e) => e.rpcUrl === url);
 
     if (!isValidUrl && url !== '') {
-      this.setErrorTo(
-        stateKey,
-        this.context.t(
-          this.isValidWhenAppended(url) ? 'urlErrorMsg' : 'invalidRPC',
-        ),
-      );
+      const errorKey = this.isValidWhenAppended(url)
+        ? 'urlErrorMsg'
+        : 'invalidRPC';
+      this.setErrorTo(stateKey, {
+        key: errorKey,
+        msg: t(errorKey),
```

Comment:
Note for tomorrow: Pass the key directly per our "pass string literals directly to `t(..)`" rule

Reply:
fixed